src/amoc_api_status_handler.erl

```diff
@@ -9,20 +9,17 @@
 -export([rest_init/2,
          allowed_methods/2,
          content_types_provided/2,
-         content_types_accepted/2,
-         process_request/2,
-         process_json/2]).
+         to_json/2,
+         get_status/0]).
 
--record(state, {action}).
+-type state() :: [].
 
--type state() :: #state{}.
-
--spec trails() -> [tuple()] .
+-spec trails() -> trails:trails().
 trails() ->
     Metadata =
         #{get =>
           #{tags => ["status"],
-            description => "Gets AMOC status, whether it is running or not",
+            description => "Gets AMOC status, whether it is running or not. Returns {\"status\": true | false}",
```

Comment:
Line 22 is too long:             description => "Gets AMOC status, whether it is running or not. Returns {\"status\": true | false}",.